actionpack/test/controller/integration_test.rb

```diff
@@ -171,12 +171,24 @@ class IntegrationTestUsesCorrectClass < ActionDispatch::IntegrationTest
   def test_integration_methods_called
     reset!
 
-    %w( get post head patch put delete ).each do |verb|
+    %w( get post head patch put delete options ).each do |verb|
       assert_nothing_raised { __send__(verb, "/") }
```

Comment:
I think it is enough passing any option to excercise the delegation instead of `IntegrationTestHasNoWarnings`.